### PR TITLE
fix: add APP_NAME_SUFFIX var

### PR DESCRIPTION
**Pull request by jayzhang**
fix: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/25476910

### CODE REVIEW

File: spfx-productivity-dashboard/teamsapp.local.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ provision:
   - uses: teamsApp/create
     with:
       # Teams app name
-      name: spfx-productivity-dashboard-${{TEAMSFX_ENV}}
+      name: spfx-productivity-dashboard${{APP_NAME_SUFFIX}}
```
Comment (HuihuiWu-Microsoft) [inline spfx-productivity-dashboard/teamsapp.local.yml:12]: Also need to update manifest file for this sample?

File: react-retail-dashboard/appPackage/manifest.local.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
         "termsOfUseUrl": "https://www.microsoft.com/en-us/servicesagreement"
     },
     "name": {
-        "short": "react-retail-dashboard-${{TEAMSFX_ENV}}",
+        "short": "react-retail-dashboard${{APP_NAME_SUFFIX}}",
```
Comment (HuihuiWu-Microsoft) [inline react-retail-dashboard/appPackage/manifest.local.json:14]: Also manifest.json file?

**Comment by HuihuiWu-Microsoft** [comment]
We also have external samples needed to be updated:
1> outlook-add-in-set-signature cc @1yefuwang1 
2> bot-proactive-messaging-teamsfx  @HuihuiWu-Microsoft 
3> ongoing integrated Microsoft Teams sample  @formulahendry

**Review by tecton — APPROVED** [review]
Failed checks in sample validation are not changed by this pull request.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**